### PR TITLE
Adjust possesive form (Author's 1999) to proper (Author 1999)

### LINKED ISSUE #10: Replace possesive form with proper noun in citation
# Description
In the sentence *Cohen's (1999) seminal paper...*, an article by Cohen (1999) is cited and should be recorded as such. Currently, the citation appears as *Cohen's 1999*, instead of *Cohen 1999*. 
This could also lead to duplicate citations being recorded if both the possessive and regular form occur.

### PR DESCRIPTION
**Pull request by Mkranj**
Resolve #10 Replace possesive form with proper noun in citation.

When both possesive and proper form are cited, the final list will include only a single, proper form citation.

### CODE REVIEW

**Comment by ghost** [comment]
Nice feature, LGTM!

File: PapersCited.py [hunk 1]
```diff
@@ -43,6 +43,8 @@ class PhrasesToChange:
       "suradnicima": "sur.",
       "suradnici": "sur.",
       "suradnika": "sur.",
+      # Change possesive form to a proper noun
+      "'s" : ""
```
Comment (krisbak1) [inline PapersCited.py:47]: I hope something like "O’Sullivan, S.B., Schmitz, T.J. and Fulk, G.D. (2014) Physical Rehabilitation. 6th Edition" will not cause troubles 😄

**Review by krisbak1 — APPROVED** [review]

File: PapersCited.py (hunk 1 above)
Comment (Mkranj) [reply PapersCited.py:47]: Good thinking. 
The replacement is case sensitive and only changes lowercase "s", so surnames such as O'Sullivan will be unaffected.
Will add a test to enforce it.

Comment (Mkranj) [reply PapersCited.py:47]: Added test that confirms it in 7172274a

**Pull request merged**